lib/telephony/pinpoint/voice_sender.rb

```diff
@@ -3,7 +3,7 @@
 module Telephony
   module Pinpoint
     class VoiceSender
-      # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/BlockLength
+      # rubocop:disable Metrics/BlockLength
```

Comment:
There's a matching `# rubocop:enable ...` below. Should it be updated to the new set of enabled rules?